### PR TITLE
Toni img centered, quotes fixed in Toni's pressRelease.

### PR DESCRIPTION
**Pull request by kumardennis**

### CODE REVIEW

File: src/styles/global.css [hunk 1]
```diff
@@ -18,6 +18,12 @@ body {
   font-family: 'Fira Sans', sans-serif;
   line-height: 1.125em;
 }
+
+body blockquote {
```
Comment (edanchenkov) [inline src/styles/global.css:27]: Don't you have this change in another branch?

**Review by edanchenkov — APPROVED** [review]

File: src/styles/global.css (hunk 1 above)
Comment (kumardennis) [reply src/styles/global.css:27]: yeah that was for another press Release in a different task, this is a different task for toni.

**Comment by edanchenkov** [comment]
@kumardennis  Merge conflict

File: src/components/Featured.js [hunk 2]
```diff
@@ -5,7 +5,7 @@ import Hex from './Hex';
 import HexImage from './HexImage';
 import FeaturedEvent from './FeaturedEvent';
 import FeaturedCase from './FeaturedCase';
-import HexPic from '../images/new-bg-square-smaller.jpg';
+import HexPic from '../images/new-bg-square-smaller2.jpg';
```
Comment (edanchenkov) [inline src/components/Featured.js]: Same here, do you really need to keep old image?

**Review by edanchenkov — CHANGES_REQUESTED** [review]
Do we really need to keep old image? See Featured.js

**Pull request merged**